index.html

```diff
@@ -1,227 +1,293 @@
+            </ul>
+          </div>
+          <div class="project">
+            <button><a href="">See project</a></button>
+          </div>
+        </div>
+      </div>
+      <div class="grid g9"></div>
+      <div class="grid g10"></div>
+      <div class="grid g2" id="inverse">
+        <div class="box2">
+          <h1>Uber Navigation</h1>
+          <div class="uber">
+            <p class="p1">UBER</p>
+            <p class="p2">Lead Developer</p>
+            <p class="p3">2018</p>
+          </div>
+          <h2>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </h2>
+          <div class="buttons">
+            <ul>
+              <li><button>html</button></li>
+              <li><button>css</button></li>
+              <li><button>javascript</button></li>
+            </ul>
+          </div>
+          <div class="project">
+            <button><a href="">See project</a></button>
+          </div>
+        </div>
+        <div class="box1">
+          <img class="image1" src="/images/imagen4.png" />
+        </div>
+      </div>
+      <div class="grid g12"></div>
+    </div>
+    <div class="about" id="about">
+      <div class="about1">
+        <p class="am">About Myself</p>
+        <p class="intro2">
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don't hesitate to
+          contact me.
+        </p>
+        <p class="lc">Lets Connect</p>
+        <div class="icons2">
+          <div class="icon1"><i class="fa-brands fa-twitter"></i></div>
+          <div class="icon2"><i class="fa-brands fa-linkedin-in"></i></div>
+          <div class="icon5"><i class="fa-regular fa-hand-peace"></i></div>
+          <div class="icon3"><i class="fa-regular fa-m"></i></div>
+          <div class="icon4"><i class="fa-brands fa-github"></i></div>
+        </div>
+        <button class="resume">Get my resume</button>
+      </div>
+      <div class="about2">
+        <div class="languajes">
+          <button class="boton"><p>Languages</p></button>
+          <div class="arrows"><p>></p></div>
+          <div class="dropmenu1">
+            <ul>
+              <li>
+                <div class="java">
+                  <i class="fa-brands fa-square-js"></i>
+                  <p>JavaScript</p>
                 </div>
-                <h2>
-                    A daily selection of privately personalized reads; no accounts or sign-ups required.
-                </h2>
-                <div class="buttons">
-                    <ul>
-                        <li><button>html</button></li>
-                        <li><button>css</button></li>
-                        <li><button>javascript</button></li>
-                    </ul>
+              </li>
+              <li>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** **Good job by fixing all the chnages mentioned by the previous reviewer.** Howerver, I noticed that there are some thing to improve again the match the Figma design. I think it would be better if you use the correct `color`, `fonts` on all elements of your project. Below is an example image of some of them:
![Screenshot from 2023-03-29 19-26-12](https://user-images.githubusercontent.com/83042742/228690572-5be9f031-d441-4f29-afb4-06198d22afce.png)

**I recommended you check everything and fix them. As you fixed all changes mentioned by the previous reviewer, that is why I mentioned this as required but I suggest you fix them before merging you pull request.**